code/modules/vueui/ui.dm

```diff
@@ -230,6 +230,11 @@ main ui datum.
   */
 /datum/vueui/Topic(href, href_list)
 	update_status()
+	if(href_list["vueuiforceresource"])
```

Comment:
`if (user != usr)` should still be checked before this. Otherwise there's a threat of someone basically DoS-ing you by constantly having the server send you data.